resources/js/components/Checkout/Checkout.vue

```diff
@@ -311,6 +311,11 @@
                 }
 
                 return this.removeUnusedAddressInfo(this.$root.checkout.billing_address)
+            },
+            currentShipmentMethod: function () {
```

Comment:
```suggestion
            currentShippingMethod: function () {
```
Magento is using "shipping" everywhere (not "shipment"), using that is more inline I think?